cli/dev/utils.go

```diff
@@ -885,10 +886,10 @@ func RemoveNetwork(client *docker.Client, name string) error {
 }
 
 func GetNodeId() string {
-	// Allow device id override if the env var is set.
+	// Allow device or node id override if the env var is set.
 	testDeviceId, _ := os.Hostname()
-	if os.Getenv(DEVTOOL_HZN_DEVICE_ID) != "" {
-		testDeviceId = os.Getenv(DEVTOOL_HZN_DEVICE_ID)
```

Comment:
We should still check for `DEVTOOL_HZN_DEVICE_ID`, if it is empty then try `GetDeviceId()`. This is because user could specify `DEVTOOL_HZN_DEVICE_ID` to something else.

Reply:
addressed in recent changes